taskcat/_cli_modules/test.py

```diff
@@ -28,6 +29,48 @@ class Test:
     Performs functional tests on CloudFormation templates.
     """
 
+    @staticmethod
+    def retry(region: str, stack_name: str, resource_name: str, config_file: str = "./.taskcat.yml",
+        project_root: str = "./"):
+        """[ALPHA] re-launches a child stack using the same parameters as previous launch
+
+        :param region: region stack is in
+        :param stack_name: name of parent stack
+        :param resource_name: logical id of child stack that will be re-launched
+        :param config_file: path to either a taskat project config file or a
+        CloudFormation template
+        :param project_root_path: root path of the project relative to input_file
+        """
+        LOG.warning("test retry is in alpha feature, use with caution")
+        project_root_path: Path = Path(project_root).expanduser().resolve()
+        input_file_path: Path = project_root_path / config_file
+        config = Config.create(
+            project_root=project_root_path,
+            project_config_path=input_file_path
+        )
+        profile = config.config.general.auth.get(region, config.config.general.auth.get("default", "default"))
+        cfn = boto3.Session(profile_name=profile).client("cloudformation", region_name=region)
```

Comment:
Why calls to boto3 when ClientFactory exists, and a region.client method available ?

Reply:
Mostly because I'm re-building everything on line 72 by directly calling `Test.run`. So there's no re-use of any clients/sessions that are consumed here. Yes, that could be a lot better, but at a non-trivial level of effort, and seeing as this is alpha, I'm not inclined to put the effort in until the feature has had a chance to bake and gain adoption.

Reply:
Thinking about the context of this function, I understand the challenges you run into here. I’m pleased that this has an ALPHA tag, because that’s clearly appropriate. 

Even in an alpha state, this is beneficial - so have an approval.